fabric-common-tags-api-v1/src/main/java/net/fabricmc/fabric/api/tags/v1/CommonEnchantmentTags.java

```diff
@@ -0,0 +1,30 @@
+package net.fabricmc.fabric.api.tags.v1;
+
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.tag.TagKey;
+
+import net.fabricmc.fabric.impl.v1.TagRegistration;
+
+public class CommonEnchantmentTags {
+	/**
+	 * A tag containing enchantments that increase the amount or quality of drops from blocks.
+	 */
+	public static final TagKey<Enchantment> FORTUNE = register("fortune");
+	/**
+	 * A tag containing enchantments that increase the amount or quality of drops from entities.
+	 */
+	public static final TagKey<Enchantment> LOOTING = register("looting");
+	/**
+	 * A tag containing enchantments that cause a block to drop itself in item form
+	 * rather than some other item, such as {@link net.minecraft.enchantment.Enchantments#SILK_TOUCH}.
+	 */
+	public static final TagKey<Enchantment> IDENTICAL_DROPS = register("identical_drops");
```

Comment:
"Changing drops" is what those tags are for...

Yes, there are mods that do duplicate their functionality.

Reply:
Looting/fortune specifically is an "increase" in drops, what about a curse that worsens it? Do these need to be seperate tags at all?

I have never once seen silk touch functionality duplicated.

What about the tags for existing common enchants like the damage boosting ones?

Reply:
I don't believe vanilla has any enchantments that make items worse besides binding or vanishing curses.
A tag has been added for damage-boosting enchantments.

Reply:
My point was more than these could be significantly more general as I don't see any reason to make it only positive ones, although at minimum I don't see a reason to separate them by accepted tool

Reply:
They're only positive because vanilla only has positive ones. They aren't separated by tool, but by target.

Reply:
Why do we care about vanilla though? This is for modders and mod compat

Reply:
Tags representing vanilla concepts and providing a naming scheme is my goal with this module, I don't know if FAPI would accept tags representing non-vanilla concepts.
Perhaps these tags need a rename to better serve this purpose.

Here's an example that's more well defined, since enchantment tags are rather new:

> We provide tags for copper, `c:copper_ores` and `c:copper_ingots`.
> Mods looking to add a new metallic ore, such as tin, can see the pattern that has been established and adopt `c:tin_ores` and `c:tin_ingots`. 

Vanilla has no tin, therefore we won't provide tags *specifically* for tin, but it can be expected for mods to use those tags as a pattern has been established. Perhaps `c:looting` should be `c:increases_entity_drops`, that way mods that add enchantments with negative effects would naturally fall into `c:decreases_entity_drops`. 

It is impractical to include every possible tag, but I hope to include a wide enough selection that additions to the "mod compat tag space" have a pattern they can be reasonably expected to follow, such that a mod making `c:ingot_tin` is the clear outlier without surveying mod tags.

Reply:
what enchantments could possibly go under identical_drops other than "just silk touch again" or "silk touch but it only works on certain blocks" or something

why, as a mod, would i ever want the ability enumerate silktouchish enchanents without actually knowing what blocks they apply to?

i don't get it